invenio_base/cli.py

```diff
@@ -80,6 +82,25 @@ def list_entrypoints(entry_point):
             click.echo('  {0}'.format(ep))
 
 
+@instance.command('migrate_secret_key')
+@click.option('--old-key', required=True)
+@with_appcontext
+def migrate_secret_key(old_key):
+    """Call entry points exposed for the SECRET_KEY change."""
+    if 'SECRET_KEY' not in current_app.config:
+        click.secho('SECRET_KEY is not set in the configuration.', fg='red')
```

Comment:
You can use a [`ClickException`](http://click.pocoo.org/5/api/#click.ClickException) instead. That way your tests will be able to test this scenario.